system-tests/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
+    "lint": "tsc --project .",
```

Comment:
I'm not seeing where this is running in CI. The mono-repo `lint-types` job execute when the mono-repo `build` job finishes, but there doesn't seem to be a lint/type script check when the `system-test-node-moduels` job finished. Am I missing it?

Reply:
Nice catch, thank you, I accidentally reverted that change. Fixed in 12d9385b1687805f7b35113c3b810c49df9e874e